modules/workspace.nix

```diff
@@ -170,5 +197,20 @@ in
         priority = 3;
       };
     })
+    (lib.mkIf (cfg.workspace.wallpaperPictureOfTheDay != null) {
+      programs.plasma.startup.desktopScript."set_wallpaper_potd" = {
+        text = ''
+          let allDesktops = desktops();
+          for (const desktop of allDesktops) {
+              var desktop = allDesktops[desktopIndex];
+              desktop.wallpaperPlugin = "org.kde.potd";
+              desktop.currentConfigGroup = Array("Wallpaper", "org.kde.potd", "General");
```

Comment:
Use array literal syntax
```js
["Wallpaper", "org.kde.potd", "General"]
```
I know that KDE docs use the Array(...) form but it's really just outdated. Writing modern JS works just fine.